src/state/queries/messages/list-conversations.tsx

```diff
@@ -152,7 +161,11 @@ export function ListConvosProviderInner({
               function filterConvoFromPage(
                 convo: ChatBskyConvoDefs.ConvoView[],
               ) {
-                return convo.filter(c => c.id !== log.convoId)
+                return convo.filter(
+                  c =>
+                    c.id !==
+                    (log as ChatBskyConvoDefs.LogCreateMessage).convoId,
```

Comment:
This is checked above, but within this nested closure TS loses it. This all runs synchronously, so I took a shortcut instead of doing the same checks again.